### PR TITLE
VPN-3159: Mac/Sentry: Upload debug symbols :) 

### PR DESCRIPTION
**Pull request by strseb**
## Description

This PR add's symbol generation and uploading to sentry for the Macos Task :)

### CODE REVIEW

**Review by mozrokafor — APPROVED** [review]
LGTM

File: src/apps/vpn/cmake/macos.cmake [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 set_target_properties(mozillavpn PROPERTIES OUTPUT_NAME "Mozilla VPN")
 
+# Force llvm to add DWARF info on Release builds
```
Comment (oskirby) [inline src/apps/vpn/cmake/macos.cmake:7]: What I think we really want to be using to get debugging symbols is to set `-DCMAKE_BUILD_TYPE=RelWithDebInfo` when invoking CMake on the command line.

**Review by oskirby — CHANGES_REQUESTED** [review]

**Comment by strseb** [comment]
@oskirby - Sorry for the long wait. It's now using `-DCMAKE_BUILD_TYPE=RelWithDebInfo`. 
While at it i moved sentry to a level-1 secret. Meaning we can dry-run this flow even on pr's :)

File: taskcluster/scripts/build/macos_build.sh [hunk 2]
```diff
@@ -67,34 +67,32 @@ print Y "Updating submodules..."
...
-    print Y "Fetching tokens..."
-    # Only on a release build we have access to those secrects. 
-    ./taskcluster/scripts/get-secret.py -s project/mozillavpn/tokens -k sentry_dsn -f sentry_dsn
-    ./taskcluster/scripts/get-secret.py -s project/mozillavpn/tokens -k sentry_envelope_endpoint -f sentry_envelope_endpoint
-    export SENTRY_ENVELOPE_ENDPOINT=$(cat sentry_envelope_endpoint)
-    export SENTRY_DSN=$(cat sentry_dsn)
+# Install dependendy got get-secret.py 
+python3 -m pip install -r taskcluster/scripts/requirements.txt
+print Y "Fetching tokens..."
+# Only on a release build we have access to those secrects.
```
Comment (oskirby) [inline taskcluster/scripts/build/macos_build.sh:73]: So, just out of curiosity, what happens when running these scripts on a PR/Debug build?

**Review by oskirby — APPROVED** [review]

File: taskcluster/scripts/build/macos_build.sh (hunk 2 above)
Comment (strseb) [reply taskcluster/scripts/build/macos_build.sh:73]: Ahh that comment is out of date. 
- Given we now have a req that all Taskcluster jobs are coming from people with repo-write access  we now have a  `project/mozillavpn/level-1` namespace for secrets, thereforce PR now can access those tokens 😅

**Pull request merged**